client/landing/gutenboarding/components/domain-picker-button/index.tsx

```diff
@@ -89,14 +88,24 @@ const DomainPickerButton: FunctionComponent< Props > = ( {
 				/>
 			) }
 			{ isDomainPopoverVisible && (
-				<Popover onClose={ handleClose } onFocusOutside={ handleClose } focusOnMount={ false }>
-					<DomainPicker
-						onDomainSelect={ handleDomainSelect }
-						onDomainPurchase={ handlePaidDomainSelect }
+				<div className="domain-picker-button__popover-container">
+					<Popover
+						className="domain-picker-button"
```

Comment:
This seems to belong to the button rather than the popover?

Reply:
It does collide with https://github.com/Automattic/wp-calypso/pull/40742/files#diff-9fb3f50c91d02f132bfab858e87cb792R70 :grimacing: